### PR TITLE
add angular material. first commit

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/app/app-material.module.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  MatInputModule,
+  MatSelectModule,
+  MatRadioModule,
+  MatDatepickerModule,
+  MatNativeDateModule,
+  MatChipsModule,
+  MatTooltipModule,
+  MatTableModule,
+  MatPaginatorModule,
+  MatCardModule
```
Comment (aatroshcenko) [inline src/app/app-material.module.ts:22]: Do we really need all of these modules?

File: src/main.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import 'hammerjs';
```
Comment (aatroshcenko) [inline src/main.ts:1]: What is the import for?

File: src/theme.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/*"styles": [
+"./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
+"src/styles.scss",
+"src/theme.scss"
+]*/
```
Comment (aatroshcenko) [inline src/theme.scss:5]: It should be removed.

File: src/app/app-material.module.ts (hunk 1 above)
Comment (MariyaSin) [reply src/app/app-material.module.ts:22]: at this stage, I do not have a clear idea of ​​what is useful and what is not

File: src/main.ts (hunk 2 above)
Comment (MariyaSin) [reply src/main.ts:1]: you're right, we don't need hamьerjs

File: src/theme.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
+@import '~@angular/material/theming';
+@include mat-core();
+
+/* ======== Angular material custom themes ======== */
+
+$theme-primary: mat-palette($mat-grey, 100);
+$theme-accent: mat-palette($mat-grey, 900);
+$app-theme: mat-light-theme($theme-accent, $theme-primary);
```
Comment (Debxit) [inline src/theme.scss:8]: image
Первым параметром идет primary цвет

File: src/theme.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+@import '~@angular/material/theming';
+@include mat-core();
+
+/* ======== Angular material custom themes ======== */
+
+$theme-primary: mat-palette($mat-grey, 100);
+$theme-accent: mat-palette($mat-grey, 900);
```
Comment (Debxit) [inline src/theme.scss:7]: акцентный цвет обычно яркий, поэтому предлагаю использовать $mat-orange, 700

**Review by Debxit — CHANGES_REQUESTED** [review]

**Review by Debxit — APPROVED** [review]

**Pull request merged**